src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,8 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String DELIMITER = "\\W+";
+    private static final String DELIMITER_FOR_RESULT = " ";
+    private static final char LETTER = 'w';
+
     public String[] readFromFile(String fileName) {
-        //write your code here
-        return null;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            StringBuilder stringBuilder = new StringBuilder();
+            String value = reader.readLine();
+            if (value == null) {
+                return new String[] {};
+            }
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+            String[] split = stringBuilder.toString().toLowerCase().split(DELIMITER);
```

Comment:
better to use nouns as names for variables
if this string were something like below it would be much more understandable:
```suggestion
            String[] fileContentByWords = fileContent.toString().toLowerCase().split(DELIMITER);
```